.github/workflows/pipeline.yml

```diff
@@ -19,6 +19,7 @@ env:
   DOCKERHUB_ORG_NAME: ${{vars.DOCKERHUB_ORG_NAME}}
   RUN_LINK_REPO: "neonlabsorg/neon-proxy.py"
   BUILD_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+  TRACER_DB_TYPE: clickhouse
```

Comment:
As I see in CI, you need to specify `TRACER_DB_TYPE` for tests environment, not for build.